xmtp_api/src/mls.rs

```diff
@@ -82,49 +82,10 @@ where
     #[tracing::instrument(level = "trace", skip_all, fields(group_id = hex::encode(&group_id)))]
     pub async fn query_group_messages(
         &self,
-        group_id: Vec<u8>,
-        id_cursor: Option<u64>,
+        group_id: GroupId,
+        cursor: Cursor,
     ) -> Result<Vec<GroupMessage>> {
-        tracing::debug!(
-            group_id = hex::encode(&group_id),
```

Comment:
paging was moved to the lower level api client implementation, since this paging strategy was v3-specific